Backend/app/spotify_electron/health/health_controller.py

```diff
@@ -1,21 +1,103 @@
 """
 Health controller for handling incoming HTTP Requests
 """
+from fastapi import APIRouter, HTTPException
+from fastapi.responses import JSONResponse
+from pydantic import BaseModel
+from typing import Dict
+from starlette.status import HTTP_200_OK, HTTP_503_SERVICE_UNAVAILABLE
 
-from fastapi import APIRouter
-from fastapi.responses import Response
-from starlette.status import HTTP_200_OK
+from app.auth.auth_schema import AuthConfig
+from app.database.DatabaseConnectionManager import DatabaseConnectionManager
+from app.spotify_electron.song.providers.song_service_provider import SongServiceProvider
+from app.logging.logging_schema import SpotifyElectronLogger
+from app.logging.logging_constants import LOGGING_HEALTH
 
 router = APIRouter(prefix="/health", tags=["health"])
+logger = SpotifyElectronLogger(LOGGING_HEALTH).getLogger()
 
+class HealthCheckResponse(BaseModel):
+    status: str
```

Comment:
Instead of status:str we can create a Enum that encapsulates different status. With that we can make the code more reusable and concise